lib/WUI/netdev.c

```diff
@@ -79,21 +81,24 @@ static void tcpip_init_done_callback(void *arg) {
         netif_set_link_callback(&eth0, netif_link_callback);
         netif_set_status_callback(&eth0, netif_status_callback);
     }
+    if (netif_add_noaddr(&wlan0, NULL, espif_init, tcpip_input)) {
+        netif_set_link_callback(&wlan0, netif_link_callback);
+        netif_set_status_callback(&wlan0, netif_status_callback);
+    }
     wui_marlin_client_init();
     osMessagePut(networkMbox_id, EVT_TCPIP_INIT_FINISHED, 0);
 }
 
 void netdev_get_ipv4_addresses(uint32_t netdev_id, lan_t *config) {
+    // TODO: deduplicate
```

Comment:
For that, maybe we should have an array of the interfaces and index by netdev_id? We already do that for some other config structures I believe.

Reply:
Actually it has some NODEV value = 2 that would be really unfortunate to use as array index by accident.